index.html

```diff
@@ -123,267 +123,277 @@ <h1>Чем мы занимаемся?</h1>
-                        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Netus eget velit quisque accumsan amet tortor. Velit, volutpat egestas fringilla mi porttitor tempus. Placerat dui.
+        <div class="wrapper">
+            <div class="fast-start">
+                    <div class="fast-start__logo">
+                        <img src="img/Ellipse%201%20(Stroke).png">
                     </div>
-                    <div class="fast-start__description-progress">
-                        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Netus eget velit quisque accumsan amet tortor. Velit, volutpat egestas fringilla mi porttitor tempus. Placerat dui.
+                    <div class="fast-start__header">
+                       <h1>Быстрый старт </h1>
                     </div>
-                    <div class="fast-start__description-progress">
-                        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Netus eget velit quisque accumsan amet tortor. Velit, volutpat egestas fringilla mi porttitor tempus. Placerat dui.
+                    <div class="fast-start__description">
+                        Больше 90% учеников прошли полный курс и смогли <br>собрать свой первый компьютер
                     </div>
-                </div>
-        </div>
-        <div class="profession wrapper">
-            <div class="profession__header">
-                Получите профессию прямо сейчас
-            </div>
-            <div class="profession__row">
-                <div class="profession__column>">
-                    <div class="profession__about">
-                        <div class="profession__icon">
-                            <img src="img/icon-1.png">
+                    <div class="fast-start__status-progress-bar">
+                        <div class="fast-start__number">
+                            100%
+                        </div>
+                        <div class="fast-start__number">
+                            75%
+                        </div>
+                        <div class="fast-start__number">
+                            50%
                         </div>
-                        <div class="profession__title">Только практические<br> навыки в работе</div>
-                        <div class="profession__text">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Netus eget velit quisque accumsan amet tortor. Velit, volutpat egestas fringilla mi porttitor tempus. Placerat dui.
+                        <div class="fast-start__number">
+                            Итог
                         </div>
                     </div>
-                </div>
-                <div class="profession__column">
-                    <div class="profession__about">
-                        <div class="profession__icon">
-                            <img src="img/icon-2.png">
+                    <div class="fast-start__progress-bar fast-start__progress-bar-xl">
```

Comment:
`fast-start__progress-bar` - it is element, `fast-start__progress-bar_xl` - it is a modifier. Use underscore, please. Fix it everywhere on the page.